src/spdx.ts

```diff
@@ -0,0 +1,56 @@
+import * as spdxlib from '@onebeyond/spdx-license-satisfies'
+import parse from 'spdx-expression-parse'
+
+/*
+ * NOTE: spdx-license-satisfies methods depend on spdx-expression-parse
+ * which throws errors in the presence of any syntax trouble, unknown
+ * license tokens, case sensitivity problems etc. to simplify handling
+ * you should pre-screen inputs to the satisfies* methods using isValid
+ */
+
+// accepts a pair of well-formed SPDX expressions. the
+// candidate is tested against the constraint
+export function satisfies(
+  candidateExpr: string,
+  constraintExpr: string
+): boolean {
+  try {
+    return spdxlib.satisfies(candidateExpr, constraintExpr)
+  } catch (_) {
+    return false
+  }
+}
+
+// accepts an SPDX expression and a non-empty list of licenses (not expressions)
+export function satisfiesAny(
+  candidateExpr: string,
+  licenses: string[]
+): boolean {
+  try {
+    return spdxlib.satisfiesAny(candidateExpr, licenses)
+  } catch (_) {
+    return false
+  }
+}
+
+// accepts an SPDX expression and a non-empty list of licenses (not expressions)
+export function satisfiesAll(
+  candidateExpr: string,
+  licenses: string[]
+): boolean {
+  try {
+    return spdxlib.satisfiesAll(candidateExpr, licenses)
+  } catch (_) {
+    return false
+  }
+}
```

Comment:
☝️ The conditions that throw an error in the new SPDX parsing and eval are very strict, and this made testing and using the library more awkward, so I moved to just return `false` in these cases, and in cases where error bubble up was a sentinel in the logic, I moved to apply an explicit validity check on the inputs instead.

I can go into more detail if desired about why Jest didn't play nice with the error bubble up previously for those that I didn't chat with about this already, but I'm pretty happy with the change so far. Let me know if you disagree or want to sync about it though 👍